eos/form-factors/analytic-b-to-gamma-qcdf.hh

```diff
@@ -0,0 +1,107 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Stephan Kuerten
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_SRC_FORM_FACTORS_ANALYTIC_B_TO_GAMMA_QCDF_HH
+#define EOS_GUARD_SRC_FORM_FACTORS_ANALYTIC_B_TO_GAMMA_QCDF_HH 1
+
+#include <eos/form-factors/b-lcdas.hh>
+#include <eos/form-factors/mesonic.hh>
+#include <eos/utils/diagnostics.hh>
+#include <eos/utils/parameters.hh>
+#include <eos/utils/options.hh>
+#include <eos/utils/qualified-name.hh>
+
+#include <array>
+#include <string>
+#include <memory>
+
+namespace eos
+{
+    class AnalyticFormFactorBToGammaQCDF:
+        public FormFactors<PToGamma>
+    {
+        private:
+
+            double mu = 1.3; // TODO: change for UsedParameter?
```

Comment:
Yes, change to parameter.

Reply:
Do you have a suggestion for the parameter naming scheme to be used in `eos/parameters/hadronic-matrix-elements.yaml`?

Reply:
``B_u->gamma::mu@QCDF``?